misc/msr-lock.md

```diff
@@ -25,60 +23,96 @@ So to fix it we have 2 options:
   
 Note: Penyrn based machines actually don't need to worry about unlocking this register
 
+## Checking if your firmware supports CFG Lock unlocking
+
+Before proceeding with the reading of this guide, first of all you'd need to check if your firmware supports CFG Lock unlocking.
+To check it, you can proceed into two ways:
+
+1. use the DEBUG version of OpenCore and check what the log says about CFG Lock
+2. use a tool called `VerifyMsrE2` which will speed up the whole checking process
+
+Personally I prefer using the first choice, but to avoid confusion I'll choose option 2.
```

Comment:
Could you elaborate with the OpenCore logs as well? Would be easier for the users to know what they should be looking for in their logs